.ahoy.yml

```
@@ -36,9 +36,47 @@ commands:
+    usage: Up the docker containers
+    cmd: |
+      clear
+      npm run build
+      docker compose up -d
+
+  stop:
+    usage: Down the docker containers
+    cmd: |
+      clear
+      docker compose down
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)